spikingjelly/activation_based/functional/misc.py

```
@@ -13,9 +16,338 @@
+
+    * **English**
+
+    Fuse adjacent ``Conv*`` and ``BatchNorm*`` modules in an evaluation-mode model
+    into a single convolution module. Both native ``torch.nn`` layers and
+    SpikingJelly activation-based ``layer.Conv*`` / ``layer.BatchNorm*`` wrappers
+    are supported.
+
+    The input model must be in ``eval()`` mode. The returned value is a fused
+    ``fx.GraphModule``.
+
+    :param net: EN: Evaluation-mode module to transform. 中文：待变换的评估模式模型，必须已经调用 ``eval()``。
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF002)